### DISCUSSION TITLE
Custom-Field in Netbox

### DISCUSSION THREAD

**Discussion by BoxxyCore** [netbox-community/netbox#11596] [Q&A]
Hello everyone,
I want to add Information from an CSV File to an Costum field which is named cf_hwn there i want to add a server but everytime when i want to add Servers via Import than i get the following  Message:

**Row 1 cf_hwn: Enter a list of values.**

The CSV-Options looks like 

![image](https://user-images.githubusercontent.com/123652097/214873614-cffadf5e-aee5-4be9-9f0f-b5d58140ebd5.png)

**Comment by candlerb** [comment]
This is a custom field on Virtualization>VirtualMachine, is that correct?

What *type* did you define this custom field as? (e.g. text, selection, object, multiple objects)?

Can you show an example of the CSV data you are trying to upload, including the column headings?

**Reply by BoxxyCore** [reply]
Sorry i forgot to reply for the Question to the CSV 

Here: 
name,status,site,cluster,device,tenant,cf_am
namexy,prod,dc01,cluster01,hwn99,tenantxy

**Reply by candlerb** [reply]
You have 7 items in the column header, but only 6 in the data row.  At very least (if you want to set cf_am to empty) you should add a comma to the end of the data row.

**Reply by BoxxyCore** [reply]
My Fault this is the right one: 

name,status,site,cluster,device,tenant,cf_am
namexy,prod,dc01,cluster01,hwn99,tenantxy,am09

**Comment by BoxxyCore** [comment]
This is the Configuration from the Costum-Field
![image](https://user-images.githubusercontent.com/123652097/215053370-37429738-48f1-43df-bf44-ee0eba7c73ec.png)

The Type of this Field is "Multiple objects (Virtualmachine)"

And Thanks for Your Answer :)

**Reply by candlerb** [reply]
"Multiple objects" allows 0, 1 or more items to be selected, and hence you must provide a _list_ of objects (as your error message says).  I don't know how to provide a list within a CSV column, or even if it's possible at all in Netbox.  You may have to use the JSON/YAML import capability instead (newly added in v3.4.x)

![image](https://user-images.githubusercontent.com/44789/215059210-1bca114c-fc67-4acc-9fcb-96bac1aef544.png)

However, your label of "Virtual Server" (singular) suggests that you'll only ever want to select a single machine.  In that case, you could change the custom field to "Object" instead of "Multiple Objects".  Beware that changing the type of an existing field in this way will probably lead to errors in the stored data.  Better to delete the field and recreate it with the right type - although that will lose any existing data.

**Reply by BoxxyCore** [reply]
I will try it ASAP

Thanks :)

**Comment by BoxxyCore** [comment]
When i do that the following is comming: 
![image](https://user-images.githubusercontent.com/123652097/215067100-cad4d304-9e87-4215-8c3d-137ee4d0fd61.png)

**Reply by candlerb** [reply]
As I requested before: show the actual CSV data you are uploading (column headers plus one row of data)

**Reply by BoxxyCore** [reply]
Done.
